node_modules/oae-content/tests/test-content.js

```diff
@@ -3160,17 +3161,34 @@ describe('Content', function() {
         });
 
         /**
-         * Test that verifies collabdoc editors can't change permissions, but can share
+         * Test that verifies collabdoc editors can't add managers
          */
-        it('verify collabdoc editors can\'t change permissions', function(callback) {
+        it('verify collabdoc editors have limited permission changing ability', function(callback) {
             setUpUsers(function(contexts) {
-                RestAPI.Content.createCollabDoc(contexts['nicolaas'].restContext, 'Test CollabDoc', 'Doc description', 'public', [], [contexts['stuart'].user.id], [], [], function(err, contentObj) {
+                RestAPI.Content.createCollabDoc(contexts['nicolaas'].restContext, 'Test CollabDoc', 'Doc description', 'public', [contexts['dave'].user.id], [contexts['stuart'].user.id], [], [], function(err, contentObj) {
                     assert.ok(!err);
                     var members = {};
                     members[contexts['anthony'].user.id] = 'viewer';
-                    // Editor can't add new members
-                    ContentTestUtil.assertUpdateContentMembersFails(contexts['nicolaas'].restContext, contexts['stuart'].restContext, contentObj.id, members, 401, function() {
-                        ContentTestUtil.assertShareContentSucceeds(contexts['nicolaas'].restContext, contexts['stuart'].restContext, contentObj.id, _.keys(members), callback);
+                    members[contexts['ian'].user.id] = 'editor';
+                    // Editor can add new viewers and editors
+                    ContentTestUtil.assertUpdateContentMembersSucceeds(contexts['nicolaas'].restContext, contexts['stuart'].restContext, contentObj.id, members, function() {
+                        members = {};
+                        members[contexts['anthony'].user.id] = 'manager';
+                        // Editor cannot promote a user to manager
+                        ContentTestUtil.assertUpdateContentMembersFails(contexts['nicolaas'].restContext, contexts['stuart'].  restContext, contentObj.id, members, 401, function() {
+                            // Editor cannot remove anyone
```

Comment:
Do they really need to add new editors, then? That's considered managing I would think. Maybe aside from editing, they are just limited to the regular viewer permissions such as share (only if the document isn't private), etc...